eng/templates/stages/release.yml

```diff
@@ -390,7 +390,11 @@ stages:
               target_branch="${target_branch#internal/}"
             fi
           fi
-
+          
+          # The resolved variables will be utilized to update the Versions.props file within the submit-source-build-release-pr script.
+          resolvedSourceBuiltArtifactsFileName=$(basename $(PIPELINE.WORKSPACE)/$(sourceBuiltArtifactsFileName))
+          resolvedSdkArtifactFileName=$(basename $(PIPELINE.WORKSPACE)/$(sdkArtifactFileName))
```

Comment:
```suggestion
          resolvedSourceBuiltArtifactsFileName=$(basename "$(PIPELINE.WORKSPACE)/$(sourceBuiltArtifactsFileName)")
          resolvedSdkArtifactFileName=$(basename "$(PIPELINE.WORKSPACE)/$(sdkArtifactFileName)")
```

Reply:
Perhaps I am missing something. However, I am not obtaining the desired outcome in this manner. 
"$(PIPELINE.WORKSPACE)/$(sourceBuiltArtifactsFileName)")
https://dev.azure.com/dnceng/internal/_build/results?buildId=2186015&view=logs&j=19992227-62fb-5b50-4e29-3b72bd33eea1&t=9281c869-17dc-5ef7-6bc6-1c1da071debd&l=103
it returns: Private.SourceBuilt.Artifacts.*.centos.8-x64.tar.gz

Reply:
Right, the wildcard doesn't get evaluated if it's in quotes.